test/accounts.test.js

```diff
@@ -1257,29 +1258,98 @@ describe('vouching cbp earning', async assert => {
   console.log('reset accounts')
   await contracts.accounts.reset({ authorization: `${accounts}@active` })
 
+  console.log('reset settings')
+  await contracts.accounts.reset({ authorization: `${accounts}@active` })
+
+  console.log('configure rewards')
+
+  const cbpRewardResident = 5
+  const cbpRewardCitizen = 7
+
+  await contracts.settings.configure("ref.cbp1.ind", cbpRewardResident, { authorization: `${settings}@active` })
+  await contracts.settings.configure("ref.cbp2.ind", cbpRewardCitizen, { authorization: `${settings}@active` })
+
   console.log('add users')
   await contracts.accounts.adduser(firstuser, 'First user', "individual", { authorization: `${accounts}@active` })
   await contracts.accounts.adduser(seconduser, 'Second user', "individual", { authorization: `${accounts}@active` })
 
-  await contracts.accounts.testsetrs(seconduser, 50, { authorization: `${accounts}@active` })
+  console.log("seconduser is sponsor for first user")
+  await contracts.accounts.addref(seconduser, firstuser, { authorization: `${accounts}@api` })
 
-  console.log('seconduser becomes a resident and can vouch')
-  await contracts.accounts.testresident(seconduser, { authorization: `${accounts}@active` })
+  console.log("firstuser becomes resident")
+  await contracts.accounts.testresident(firstuser, { authorization: `${accounts}@active` })
 
-  console.log('vouch for first user')
-  await contracts.accounts.vouch(seconduser, firstuser, { authorization: `${seconduser}@active` })
+  const cbsAfterResident = await getTableRows({
+    code: accounts,
+    scope: accounts,
+    table: 'cbs',
+    json: true
+  })
 
-  const vouchTable = await getTableRows({
+  // console.log("cbs "+JSON.stringify(cbsAfterResident, null, 2))
+
+  assert({
+    given: 'firstuser became resident',
+    should: 'sponsor received enough cbp',
+    actual: cbsAfterResident.rows[0].community_building_score,
+    expected: cbpRewardResident
+  })
+
+  console.log("firstuser becomes citizen")
+  await contracts.accounts.testcitizen(firstuser, { authorization: `${accounts}@active` })
+  const expected_cbp_cit = await get_settings("ref.cbp2.ind")
+
+  const cbsAfterCitizen = await getTableRows({
     code: accounts,
     scope: accounts,
     table: 'cbs',
     json: true
   })
 
+  // console.log("cbs "+JSON.stringify(cbsAfterCitizen, null, 2))
+
+  assert({
+    given: 'firstuser became citizen',
+    should: 'sponsor received enough cbp',
+    actual: cbsAfterCitizen.rows[0].community_building_score,
+    expected: cbpRewardResident + cbpRewardCitizen
+  })
+
+})
+
+describe.only('Referral cbp reward organization', async assert => {
+
+  if (!isLocal()) {
+    console.log("only run unit tests on local - don't reset accounts on mainnet or testnet")
+    return
+  }
+
+  const contracts = await initContracts({ settings, accounts })
+
+  const orguser = "org1"
```

Comment:
This might not be so important but this user is not a default user in the `scripts/seeds init` command, so its account is not being created in the local node by default.

Reply:
Oh you're right... I must have forgotten to check that in

    TRANSACTION ERROR: Error: assertion failure with message: no account
    Error on actions: [{"account":"accts.seeds","name":"adduser","authorization":[{"actor":"accts.seeds","permission":"active"}],"data":{"account":"org1","nickname":"org 1","type":"organisation"}},{"account":"policy.seeds","name":"create","authorization":[{"actor":"policy.seeds","permission":"active"}],"data":{"account":"policy.seeds","backend_user_id":"nh","device_id":"nh","signature":"","policy":""}}]    ✖  Error: assertion failure with message: no account

Reply:
Added